numismatic/feeds/bravenewcoin.py

```diff
@@ -18,16 +18,29 @@ def get_list(self):
       data = self._make_request(api_url, headers=self.headers)
       response = []
 
-      for json_dict in data['digital_currencies']:
-        for key, value in json_dict.items():
-          response.insert(0, key)
+      response = [key 
+                  for json_dict in data['digital_currencies']
+                  for key, value in json_dict.items()]
 
       return response
 
     #TODO implement this
     def get_info(self, assets):
       raise NotImplementedError('Not implemented yet') 
 
-    #TODO implement this
     def get_prices(self, assets, currencies):
-      raise NotImplementedError('Not implemented yet')
+      """Not the most efficient method as BNC does not
+      allow a list for input. So we iterate over each
+      asset and currency and make separate calls"""
+      response = []
+      for asset in assets.split(','):
+        for currency in currencies.split(','):
+          api_url = f'{self.api_url}/ticker/?coin={asset}&show={currency}'
+          data = self._make_request(api_url, headers=self.headers)
+          if data['success'] != True:
+            continue
+
+          response.insert(len(response), 
```

Comment:
Rather use `response.append(...)` instead of `response.insert(len(response), ...)`